src/azure-cli/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -115,6 +130,45 @@ def process_secret_set_namespace(cmd, namespace):
     namespace.value = content
 
 
+def process_storage_uri(ns):
+    if not ns.storage_resource_uri:
+        if ns.storage_account_name and ns.blob_container_name:
+            ns.storage_resource_uri = 'https://{}.blob.core.windows.net/{}'.format(
```

Comment:
`core.windows.net` is storage endpoint for public cloud. Are you sure it's a fixed value here ?